src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,132 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    public static class Node<T> {
```

Comment:
All inner classes should be at the end of class
```suggestion
    private class Node<T> {
```